integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/DatabaseCreateAction.java

```diff
@@ -8,22 +8,22 @@
 import org.slf4j.LoggerFactory;
 
 import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
-import com.sequenceiq.it.cloudbreak.newway.RdsConfigEntity;
+import com.sequenceiq.it.cloudbreak.newway.entity.DatabaseEntity;
 import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
 
-public class RdsConfigPostAction implements Action<RdsConfigEntity> {
+public class DatabaseCreateAction implements Action<DatabaseEntity> {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(RdsConfigPostAction.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseCreateAction.class);
 
     @Override
-    public RdsConfigEntity action(TestContext testContext, RdsConfigEntity entity, CloudbreakClient client) throws Exception {
+    public DatabaseEntity action(TestContext testContext, DatabaseEntity entity, CloudbreakClient client) throws Exception {
         log(LOGGER, format(" Name: %s", entity.getRequest().getName()));
-        logJSON(LOGGER, format(" RDS post request:%n"), entity.getRequest());
+        logJSON(LOGGER, format(" Database create request:%n"), entity.getRequest());
         entity.setResponse(
                 client.getCloudbreakClient()
                         .databaseV4Endpoint()
                         .create(client.getWorkspaceId(), entity.getRequest()));
-        logJSON(LOGGER, format(" RDS created  successfully:%n"), entity.getResponse());
+        logJSON(LOGGER, format(" Database created  successfully:%n"), entity.getResponse());
```

Comment:
I know it's a minor thing but can we fix this formatting issue? I can see it's reproducing itself because no one cares.

" Database created  successfully:%n" --> " Database created successfully:%n"
(remove unnecessary seconds space between created and successfully)

By the way what is %n in String.format?

Reply:
%n means new line

Reply:
thx. but then why not simply " Database created successfully:%n" --> " Database created successfully:\n" without String.format?

Reply:
¯\_(ツ)_/¯

Reply:
If there is no reason for String.format, then please also remove that